docs/running-on-kubernetes.md

```diff
@@ -123,6 +123,20 @@ Starting with Spark 2.4.0, it is possible to run Spark applications on Kubernete
 runs in client mode, the driver can run inside a pod or on a physical host. When running an application in client mode,
 it is recommended to account for the following factors:
 
+### Make a distribution
+
+In the case you are working with latest code directly from the git code base of Apache Spark, by building from the source.
```

Comment:
Nit: `the latest code` and `a docker repository` below. Also please use `Spark` with a capital `S` consistently.

Reply:
This and the sentence below aren't quite grammatical. This also kind of duplicates the docs on how to build a release. Why not just say something like:

When testing against a custom build of Spark, from the latest source code or custom modifications, it's necessary to create a release and deploy images to a Docker repository. See the [Docker images] section, and [docs on releasing]. Example:

... example command
```